routes/clover_base.rb

```diff
@@ -65,10 +65,10 @@ def fetch_location_based_prices(*resource_types)
     # that users won't see higher price in their invoice compared
     # to price calculator and also we charge same amount no matter
     # the number of days in a given month.
-    BillingRate.rates.filter { resource_types.include?(_1["resource_type"]) }
+    BillingRate.rates.filter { resource_types.include?(_1["resource_type"]) }.sort_by { _1["active_from"] }
```

Comment:
We already return the latest billing rate, because duplicate entries overwrite each other while generating the hash below and thanks to sort, latest billing rate performs the latest overwrite.

However, I understand this is not obvious at a first glance. So I'm adding an explicit filter here.